### PR TITLE
Add Droidcon Vietnam 2023

### PR DESCRIPTION
**Pull request by colinmarsch**

### CODE REVIEW

File: _conferences/droidcon-vietnam-2023.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: "Droidcon"
+website: https://www.vietnam.droidcon.com/
+location: Ho Chi Minh City, Vietnam
+
+date_start: 2023-07-15
+date_end:   2023-07-16
```
Comment (cketti) [inline _conferences/droidcon-vietnam-2023.md]: According to the website it's a one day event. So `date_end` should be `2023-07-15` as well.

Comment (colinmarsch) [reply _conferences/droidcon-vietnam-2023.md]: Good catch, I just copied that part off the Sessionize page which might be representing the conference wrong

Comment (cketti) [reply _conferences/droidcon-vietnam-2023.md]: I just talked to one of the organizers and they updated the Sessionize page.

**Comment by cketti** [comment]
Thanks :+1:

**Pull request merged**